make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -431,92 +431,43 @@ endif
 
 ###########################################################################
 
+
 ifeq ($(USE_EXTERNAL_HARFBUZZ), true)
-  LIBHARFBUZZ_LIBS := $(HARFBUZZ_LIBS)
+   LIBFONTMANAGER_EXTRA_SRC =
+   BUILD_LIBFONTMANAGER_FONTLIB += $(LIBHARFBUZZ_LIBS)
 else
-
-  # This is better than adding EXPORT_ALL_SYMBOLS
-  ifneq ($(filter $(TOOLCHAIN_TYPE), gcc clang), )
-    HARFBUZZ_CFLAGS += -DHB_EXTERN=__attribute__\(\(visibility\(\"default\"\)\)\)
-  else ifeq ($(TOOLCHAIN_TYPE), microsoft)
-    HARFBUZZ_CFLAGS += -DHB_EXTERN=__declspec\(dllexport\)
-  endif
-
-  ifeq ($(call isTargetOs, windows), false)
-    HARFBUZZ_CFLAGS += -DGETPAGESIZE -DHAVE_MPROTECT -DHAVE_PTHREAD \
-                      -DHAVE_SYSCONF -DHAVE_SYS_MMAN_H -DHAVE_UNISTD_H \
-                      -DHB_NO_PRAGMA_GCC_DIAGNOSTIC
-  endif
-  ifeq ($(call isTargetOs, linux macosx), true)
-    HARFBUZZ_CFLAGS += -DHAVE_INTEL_ATOMIC_PRIMITIVES
-  endif
-  ifeq ($(call isTargetOs, macosx), true)
-    HARFBUZZ_CFLAGS += -DHAVE_CORETEXT
-  endif
-  ifeq ($(call isTargetOs, macosx), false)
-    LIBHARFBUZZ_EXCLUDE_FILES += libharfbuzz/hb-coretext.cc
-  endif
-  # hb-ft.cc is not presently needed, and requires freetype 2.4.2 or later.
-  LIBHARFBUZZ_EXCLUDE_FILES += libharfbuzz/hb-ft.cc
-
-  LIBHARFBUZZ_CFLAGS += $(HARFBUZZ_CFLAGS)
-
-  # For use by libfontmanager:
-  ifeq ($(call isTargetOs, windows), true)
-    LIBHARFBUZZ_LIBS := $(SUPPORT_OUTPUTDIR)/native/$(MODULE)/libharfbuzz/harfbuzz.lib
-  else
-    LIBHARFBUZZ_LIBS := -lharfbuzz
-  endif
-
-  LIBHARFBUZZ_EXTRA_HEADER_DIRS := \
-    libharfbuzz/hb-ucdn \
-    #
-
-  LIBHARFBUZZ_OPTIMIZATION := HIGH
-  # Early re-canonizing has to be disabled to workaround an internal XlC compiler error
-  # when building libharfbuzz
-  ifeq ($(call isTargetOs, aix), true)
-    LIBHARFBUZZ_CFLAGS += -qdebug=necan
```

Comment:
I couldn't find the `-qdebug=necan` in the updated makefile. Does the [JDK-8258484](https://bugs.openjdk.java.net/browse/JDK-8258484) handled is some other way?

Reply:
I'll restore this.